### DISCUSSION TITLE
Seeking tutorial to learn ImageMagick

### DISCUSSION THREAD

**Discussion by satimis** [ImageMagick/ImageMagick#5213] [Help]
Hi all,

Please advise where can I find a tutorial learning annotates

Annotated List of Command-line Options
https://imagemagick.org/script/command-line-options.php

Preferable with photos demonstrating the result of the command lines

Thanks

Regards

**Comment by fmw42** [comment]
For novices, see

http://www.imagemagick.org/discourse-server/viewtopic.php?f=1&t=9620
http://www.imagemagick.org/script/command-line-processing.php
https://imagemagick.org/script/command-line-options.php
http://www.imagemagick.org/Usage/reference.html
http://www.imagemagick.org/Usage/
https://github.com/ImageMagick/usage-markdownhttps://imagemagick.org/script/porting.php#cli
https://legacy.imagemagick.org/discourse-server/index.php

See in particular:

https://imagemagick.org/Usage/text/
https://imagemagick.org/Usage/fonts/
https://imagemagick.org/Usage/annotating/

**Reply by satimis** [reply]
Hi

Thanks for your help and your suggested URL

What I expect to find are tutorials and documents explaining the results of running IM commands and using their annotates.  What will be the result of the images?

I have the complete list of;
Annotated List of Command-line Options - ImageMagick
https://imagemagick.org/script/command-line-options.php

How to use them to run "magick" command?  What will be the resultant images?  How do they look finally?

I'm running IM command lines to edit images in bulk, not editing 1 or 2 images.  I'll copy the image files on a folder and run one IM command line to edit all of them.

Thanks

Regards

**Reply by fmw42** [reply]
There is no tutorial that directly shows each of those commands. The closest is https://www.imagemagick.org/Usage/reference.html
